pandas/tests/test_expressions.py

```diff
@@ -22,6 +22,13 @@
 
 _frame = DataFrame(randn(10000, 4), columns=list('ABCD'), dtype='float64')
 _frame2 = DataFrame(randn(100, 4), columns=list('ABCD'), dtype='float64')
+_issued_frame = DataFrame([[0, 1, 2, 'aa'], [0, 1, 2, 'aa'],
```

Comment:
these are only used once, so just construct them inside the test

Reply:
@jreback if i construct them on @pytest.mark.parametrize's argument, code is so long. is it okay?
example)
    @pytest.mark.parametrize("test_input,expected", [
        (DataFrame([[0, 1, 2, 'aa'], [0, 1, 2, 'aa'],
                    [0, 1, 5, 'bb'], [0, 1, 5, 'bb'],
                    [0, 1, 5, 'bb']], columns=['a', 'b', 'c', 'dtype']).
         loc[:, ['a', 'dtype']].
         ne(DataFrame([[0, 1, 2, 'aa'], [0, 1, 2, 'aa'],
                       [0, 1, 5, 'bb'], [0, 1, 5, 'bb'],
                       [0, 1, 5, 'bb']], columns=['a', 'b', 'c', 'dtype']).
            loc[:, ['a', 'dtype']]),
         DataFrame([[False, False], [False, False],
                    [False, False], [False, False],
                    [False, False]], columns=['a', 'dtype'])),

so, could you give me some tip to good code for this case?

Reply:
@jreback i remove them. and construct in the test. thank you